README.md

```diff
@@ -4,76 +4,42 @@ Symfony Demo Application
 The "Symfony Demo Application" is a reference application created to show how
 to develop Symfony applications following the recommended best practices.
 
-[![Build Status](https://travis-ci.org/symfony/symfony-demo.svg?branch=master)](https://travis-ci.org/symfony/symfony-demo)
-
 Requirements
 ------------
 
   * PHP 7.1.3 or higher;
   * PDO-SQLite PHP extension enabled;
   * and the [usual Symfony application requirements](https://symfony.com/doc/current/reference/requirements.html).
 
-If unsure about meeting these requirements, download the demo application and
-browse the `http://localhost:8000/config.php` script to get more detailed
-information.
-
 Installation
 ------------
 
-[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
-
-First, install the [Symfony Installer](https://github.com/symfony/symfony-installer)
-if you haven't already. Then, install the Symfony Demo Application executing
-this command anywhere in your system:
+Execute this command to install the project:
 
 ```bash
-$ symfony demo
-
-# if you're using Windows:
-$ php symfony demo
+$ composer create symfony/symfony-demo
```

Comment:
I suggest using the actual name (`create-project`) rather than relying on the symfony/console shortcut feature. If someone has a global plugin installed providing a `create` command, this would run the wrong command.